datahub-web-react/src/app/onboarding/onboarding.tsx

```diff
@@ -0,0 +1,693 @@
+                    <strong>Groups</strong> are collections of users which can be used to assign ownership to assets and
+                    configure authorization.
+                </p>
+                <p>
+                    <strong>Groups</strong> can be created natively via the DataHub UI, or synced from your Identity
+                    Provider.
+                </p>
+                <p>
+                    Learn more about <strong>Groups</strong>{' '}
+                    <a
+                        target="_blank"
+                        rel="noreferrer noopener"
+                        href="https://datahubproject.io/docs/authorization/groups"
+                    >
+                        {' '}
+                        here.{' '}
+                    </a>
+                </p>
+            </Typography.Paragraph>
+        ),
+    },
+    {
+        id: GROUPS_CREATE_GROUP_ID,
+        selector: `#${GROUPS_CREATE_GROUP_ID}`,
+        title: 'Create a new Group',
+        content: (
+            <Typography.Paragraph>
+                <p>
+                    Click here to create a new <strong>Group</strong>.
+                </p>
+            </Typography.Paragraph>
+        ),
+    },
+    {
+        id: ROLES_INTRO_ID,
+        title: 'Roles Page',
+        content: (
+            <Typography.Paragraph>
+                <p>
+                    Welcome to the <strong>Roles</strong> page!
+                </p>
+                <p>
+                    DataHub currently supports three out-of-the-box Roles: <strong>Admin</strong>,{' '}
+                    <strong>Editor</strong> and <strong>Reader</strong>.
+                </p>
+                <p>
+                    <strong>Roles</strong> are the recommended way to manage permissions on DataHub.
+                </p>
+                <p>
+                    Learn more about <strong>Roles</strong> and the different permissions for each Role{' '}
+                    <a
+                        target="_blank"
+                        rel="noreferrer noopener"
+                        href="https://datahubproject.io/docs/authorization/roles"
+                    >
+                        {' '}
+                        here.{' '}
+                    </a>
+                </p>
+            </Typography.Paragraph>
+        ),
+    },
+    {
+        id: POLICIES_INTRO_ID,
+        title: 'Policies Page',
+        content: (
+            <Typography.Paragraph>
+                <p>
+                    Welcome to the <strong>Policies</strong> page!
+                </p>
+                <p>
+                    <strong>Policies</strong> allow you to define fine-grained access controls for your DataHub
+                    instance..
+                </p>
+                <p>
+                    For most users, <strong>Roles</strong> are the recommended way to manage permissions on DataHub. If
+                    Roles do not fit your use case, then Policies can be used.
```

Comment:
"If you need fine-grained control of user access, then Policies will do the trick."

Reply:
Good call